app/policies/feed_policy.rb

```diff
@@ -16,6 +16,6 @@ def update?
   end
 
   def destroy?
-    update?
+    resource.custom? && update?
```

Comment:
This logic used to be in the view, but is more testable in the policy.  Don't allow deleting your default feed.